conformance/packages/dns-test/src/tshark.rs

```diff
@@ -226,6 +226,23 @@ impl Message {
 
         None
     }
+
+    pub fn is_rd_flag_set(&self) -> bool {
+        let Some(recdesired) = self.inner["dns.flags_tree"]
```

Comment:
Nit: I'd prefer for the Rust identifiers to match Rust naming style, like `rec_desired` (at least) or maybe even just `recursion_desired`, or go the other direction and consistently abbreviate as `rd`.